api-rest/src/test/java/com/sitionix/bffssox/controller/UserControllerTest.java

```diff
@@ -0,0 +1,67 @@
+package com.sitionix.bffssox.controller;
+
+import com.app_afesox.bffssox.api_first.dto.UserDTO;
+import com.app_afesox.bffssox.api_first.dto.UserResponseDTO;
+import com.sitionix.bffssox.domain.User;
+import com.sitionix.bffssox.mapper.UserApiMapper;
+import com.sitionix.bffssox.usecase.CreateUser;
+import org.junit.jupiter.api.DisplayNameGeneration;
+import org.junit.jupiter.api.DisplayNameGenerator;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.TestConfiguration;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.context.annotation.Bean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.when;
+
+@ExtendWith(SpringExtension.class)
+@DisplayNameGeneration(DisplayNameGenerator.ReplaceUnderscores.class)
+class UserControllerTest {
+
+    @TestConfiguration
+    static class TestContextConfiguration {
+        @Bean
+        public UserController userController(final UserApiMapper userApiMapper, final CreateUser createUser) {
+            return new UserController(userApiMapper, createUser);
+        }
+    }
+
+    @Autowired
+    private UserController userController;
+
+    @MockBean
+    private UserApiMapper userApiMapper;
+
+    @MockBean
+    private CreateUser createUser;
+
```

Comment:
pls add before each verifyNoMoreInteractions(), and everywhere where youre using mock beans